pkgs/os-specific/linux/zfs/default.nix

```
@@ -210,13 +210,14 @@ in {
 
   zfsUnstable = common {
     # check the release notes for compatible kernels
-    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.14";
+    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.15";
     latestCompatibleLinuxPackages = linuxPackages_5_13;
 
     # this package should point to a version / git revision compatible with the latest kernel release
-    version = "2.1.0";
+    version = "unstable-2021-08-30";
```

Personally I'd use
```suggestion
    version = "2.1.0+git20210830.${builtins.substring 0 7 rev}";
```
but that is of course just a matter of taste. I don't think there are any rigid guidelines in nixpkgs regarding naming of Git revisions.

---

The more common idiom in nixpkgs is `unstable-YYYY-MM-DD`.

---

date is easier to sort with nix-env.